packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -127,4 +128,24 @@ export class MigrationActions {
       throw Error(`Expected ${actual} to equal ${expected}`)
     }
   }
+
+  async assertSuccessfulConfigScaffold (configFile: `cypress.config.${'js'|'ts'}`) {
+    assert(this.ctx.currentProject)
```

Comment:
This is how I am asserting the generated `cypress.config.js` is valid. It compares the newly created one to a project that is known to work and runs in CI, so if the two configs match, we can be confident it's correct.